### DISCUSSION TITLE
Issues with: Code tutorial: Transcribe streaming audio

### DISCUSSION THREAD

**Discussion by GregStevenson** [deepgram/community#310] [General help] [answered]
This is for the .Net example, but issues may be in other examples.
1) The suggested URL in the code gives a 404. https://stream.live.vc.bbcmedia.co.uk/bbc_radio_fourlw_online_nonuk"; Was able to find it at archive.org. https://web.archive.org/web/20230308075336/https://stream.live.vc.bbcmedia.co.uk/bbc_radio_fourlw_online_nonuk
2) Using archived link was able to open socket and it will tranfer data for a while but eventually stops and HandleTranscriptReceived is always called with IsFinal being false and the Transcript.Channel is frequently null. As a result, no transcript is ever output. Not sure how to resolve this.

**Comment by jpvajda** [comment]
@GregStevenson  Can you clarify when you say the `.Net` example  are you referring to our .Net example in our `Transcribe streaming Audio Mission` in the Deepgram Console?

I do see this link in here, and wanted to confirm so we can replace this URL.

![Screenshot 2023-08-15 at 5 21 00 PM](https://github.com/deepgram/community/assets/1395158/e322a968-02c9-4b21-a8d0-7c5a53f76912)


I did check our python & node examples in this mission and they use 

http://stream.live.vc.bbcmedia.co.uk/bbc_world_service

which is a live URL you can leverage if you wish.

**Reply by jpvajda** [reply, marked as answer]
I confirmed this should be fixed very soon. Thanks for letting us know @GregStevenson
